blog/content/posts/2024-12-06-series.adoc

```diff
@@ -0,0 +1,61 @@
+---
+layout: :theme/post
+title: Want to a blog post series ?
```

Comment:
```suggestion
title: Do you want to publish a blog post series ?
```

Reply:
Obviously forgot a word...

Wanted to write : _Want to write a blog post series ?_

But will keep you suggestion.